Bump version of the pengrad's library

I noticed some exceptions during getting new updates:
`Caused by: java.lang.NumberFormatException: Expected an int but was 7661532646 at line 2 column 1871 path $.result[0].message.direct_messages_topic.topic_id at com.google.gson.stream.JsonReader.nextInt(JsonReader.java:1186) ~[gson-2.10.1.jar:na] at com.google.gson.internal.bind.TypeAdapters$7.read(TypeAdapters.java:249) ~[gson-2.10.1.jar:na] ... 39 common frames omitted`

we need to bump version of pengrad's library to the latest one

--- a/pom.xml
+++ b/pom.xml
@@ -58,8 +58,8 @@
     <properties>
         <java.version>17</java.version>
         <spring.boot.version>3.3.5</spring.boot.version>
-        <revision>0.6.0</revision>
-        <pengrad.bot.api.version>9.2.0</pengrad.bot.api.version>
+        <revision>0.5.1</revision>
+        <pengrad.bot.api.version>9.4.0</pengrad.bot.api.version>
         <spotless.plugin.version>3.0.0</spotless.plugin.version>
         <checkstyle.plugin.version>3.6.0</checkstyle.plugin.version>
         <maven.compiler.release>${java.version}</maven.compiler.release>

